client/src/components/NavBar/style.ts

```diff
@@ -20,6 +20,23 @@ export const Wrapper = styled.div`
   top: 0;
   width: 100%;
   z-index: 1;
+
+  /* Increase specificity for targeted elements specified in index.html and index_template.html */
```

Comment:
Possible to do something like instead?

```
  .bp5-button-text {
    font-family: "Inter", sans-serif;
  }
```

Reply:
Thanks for the suggestion, Ronen! 

I thought about having a smaller scope as well, but was worried that we might miss elements in the Navbar that also need Inter either now or later, thus the rationale of completing overriding the same scope of elements targeted in the `index.html` file

What do you think?

Thank you 😄

Reply:
Yes that makes sense to me! Thank you for clarifying 🙏🏽

Reply:
Awesome thanks for the great discussion 😄 💪 💪